bin/pycbc_make_html_page

```diff
@@ -224,6 +235,18 @@ for cwd in dirs:
                'dir'               : cwd,
                'dot_dot_str'       : cwd.level() * '../',
                'plots_dir'         : opts.plots_dir}
+    if local_logo:
+        if not cwd.path == "":
+            print("CWD path: " + cwd.path)
+            cwd_logo_loc = opts.output_path + cwd.path + '/logo.' + logo_ext
+            try:
+                os.symlink(logo_loc, cwd_logo_loc)
```

Comment:
Why simlink? Why not link via the HTML path directly to the one image you copied already?

Reply:
Ah, wait nevermind. Thinking about this for a few seconds, I know why it doesn't work. Would have to think of an alternative.

Reply:
I used the dot_dot_str as you suggested in slack to allow the logo to be held only in the base directory